scene/gui/base_button.cpp

```diff
@@ -466,6 +495,9 @@ void BaseButton::_bind_methods() {
 
 	BIND_ENUM_CONSTANT(ACTION_MODE_BUTTON_PRESS);
 	BIND_ENUM_CONSTANT(ACTION_MODE_BUTTON_RELEASE);
+
+	GLOBAL_DEF("gui/button/shortcut_feedback_highlight_time", 0.5);
+	ProjectSettings::get_singleton()->set_custom_property_info(PropertyInfo(Variant::FLOAT, "gui/button/shortcut_feedback_highlight_time", PROPERTY_HINT_RANGE, "0.01,10,0.01,suffix:s"));
```

Comment:
We now have
```
Variant _GLOBAL_DEF(const PropertyInfo &p_info, const Variant &p_default, bool p_restart_if_changed = false, bool p_ignore_value_in_docs = false, bool p_basic = false, bool p_internal = false);
```
so you can group both and avoid the redundant name (merged last week #68429).

```suggestion
	GLOBAL_DEF(PropertyInfo(Variant::FLOAT, "gui/button/shortcut_feedback_highlight_time", PROPERTY_HINT_RANGE, "0.01,10,0.01,suffix:s"), 0.5);
```

Reply:
And remember to sync the classref / add documentation.

Reply:
oooh nice